src/Console/Command/FixCommand.php

```diff
@@ -327,8 +327,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $config = $resolver->getConfig();
         $configFile = $resolver->getConfigFile();
 
-        if (null !== $stdErr && $configFile) {
-            $stdErr->writeln(sprintf('Loaded config from "%s".', $configFile));
+        if (null !== $stdErr) {
+            $name = $config->getName();
+            $stdErr->writeln(
+                sprintf(
+                    'Loaded config%s%s.',
+                    null === $name ? '' : ' <comment>'.$name.'</comment>',
+                    null === $configFile ? '' : ' from "'.$configFile.'"'
```

Comment:
I've added a test for this, it is the first place the name of configuration is actually used ;)